pkg/driver/controller.go

```diff
@@ -57,38 +58,20 @@ func (d *controllerService) CreateVolume(
 		return nil, status.Error(codes.InvalidArgument, "Volume Capabilities cannot be empty")
 	}
 
-	secrets := req.Secrets
-	klog.V(5).Infof("CreateVolume: Secrets contains keys %+v", reflect.ValueOf(secrets).MapKeys())
-
-	capacityBytes := int64(req.CapacityRange.GetRequiredBytes())
-
-	klog.V(5).Infof("CreateVolume: Mounting juicefs %q", req.Name)
-	jfs, err := d.juicefs.JfsMount(secrets, []string{})
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "Could not mount juicefs: %v", err)
-	}
-
-	klog.V(5).Infof("CreateVolume: Creating volume %q, capacity bytes: %d", req.Name, capacityBytes)
-	vol, err := jfs.CreateVol(req.Name, capacityBytes)
-	if status.Convert(err).Code() == codes.AlreadyExists {
-		return nil, err
-	}
-
-	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "Could not create volume %q size %d bytes: %v", req.Name, capacityBytes, err)
-	}
-
 	volume := csi.Volume{
 		VolumeId:      req.Name,
-		CapacityBytes: vol.CapacityBytes,
-		VolumeContext: req.Parameters,
+		CapacityBytes: req.CapacityRange.GetRequiredBytes(),
+		VolumeContext: map[string]string{
+			"mode":     "dynamic",
+			"capacity": strconv.FormatInt(req.CapacityRange.GetRequiredBytes(), 10),
+		},
```

Comment:
And there's no way to get the name of required PVC from PVC ID...
or k8s address will be needed here